File: ppg_sl/go/enjou.go
//言語：Go
//問題URL:https://paiza.jp/works/mondai/ppg_sl_go/ppg_sl_go__enjou
//コメント：
//コード：
//```go
package main

import (
	"fmt"
	"os"
)

func main() {
	var due int
	fmt.Scan(&due)
	if due<=3 {
		fmt.Println("BOMB")
		os.Exit(0)
	}
	fmt.Println("ONGOING")
}
//```

File: ppg_sl/javascript/enjou.js
//言語：JavaScript
//問題URL:https://paiza.jp/works/mondai/ppg_sl_js/ppg_sl_js__enjou
//コメント：
//コード：
//```js
const input = require('fs').readFileSync('/dev/stdin', 'utf8')
let due = parseInt(input, 10);
if(due<=3){
	console.log("BOMB");
	process.exit(0);
}
console.log("ONGOING");
//```